README.md

```diff
@@ -23,55 +23,12 @@ yarn
 3. Build the code and run the local version of the UI
```

Comment:
could we keep this in the readme but move it to bottom of file? i still use this any time i need to test functionality end to end, the mocks are not helpful for end to end testing

Reply:
okay adding back

Reply:
this comment was about keeping the `yarn start` info, other commend was about the view state

Reply:
done for both